### PR TITLE
added my social medias

### PR DESCRIPTION
**Pull request by danosherow**
added my linkedin and twitter to teamG-coded.md

### CODE REVIEW

File: teamG-coded.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Originals:
 - Jacory Armstrong-Jamerson | [linkedin](https://www.linkedin.com/in/jacory-armstrong-jamerson-5ab073247/)
 - Sydney Simmons
 - Destiana Clincy
-- Jon Hill
+- Jon Hill 
 - DeMarcus McMillian
-- Daniel Osherow
+- Daniel Osherow | [linked In](https://www.linkedin.com/in/daniel-osherow-309bb21a9/) | [twitter](https://twitter.com/daniel_osherow)
```
Comment (bclincy) [inline teamG-coded.md]: You were  supposed to update your linkedin so you removed the extra characters and make it your own.

**Review by bclincy — CHANGES_REQUESTED** [review]

File: teamG-coded.md (hunk 1 above)
Comment (tashianap) [inline teamG-coded.md]: need to remove the extra characters behind your last name for your linkedin url

**Review by tashianap — CHANGES_REQUESTED** [review]

File: teamG-coded.md (hunk 1 above)
Comment (danosherow) [reply teamG-coded.md]: @tashianap completed

Comment (danosherow) [reply teamG-coded.md]: @bclincy completed

**Comment by danosherow** [comment]
@bclincy changes have been made please merge

**Review by bclincy — APPROVED** [review]

**Review by tashianap — APPROVED** [review]

**Pull request merged**